### PR TITLE
Add a gcp-helpers module

### PR DESCRIPTION
**Pull request by robmorgan**
This PR adds a `gcp-helpers` module to store helper scripts that automate common GCP tasks. Initially I have included only one script:

 * `install-gcloud`: This script is meant to be run in a CircleCI job to install the latest version of the Google Cloud SDK CLI tool.

But we have the option to expand the scripts over time.

### CODE REVIEW

File: .circleci/config.yml
Comment (brikis98) [inline .circleci/config.yml:26]: NIT: Add a TODO to switch to the release version here. Got caught by this in https://github.com/gruntwork-io/terraform-aws-ci/pull/269.

File: test/README.md
Comment (brikis98) [inline test/README.md:40]: NIT: you don't really need this. Running `go test` will download dependencies automatically, if needed.

File: test/install_gcloud_test.go
Comment (brikis98) [inline test/install_gcloud_test.go:17]: Will calling this script in a test that runs in a build job running in CircleCi where we _already_ ran this script cause any issues?

Comment (robmorgan) [reply test/install_gcloud_test.go:17]: Hmmmm, well whilst it's non-ideal I guess all of the commands in the script are idempotent except adding the `apt` sources. The only way around I guess would be to run this specific test in its own isolated Docker executor which is also a bit of an overkill. Thoughts?

Comment (brikis98) [reply test/install_gcloud_test.go:17]: No strong opinion. You could run the entire test in a Docker container, or a separate CircleCi executor, or leave as-is. Your call!

Comment (robmorgan) [reply test/install_gcloud_test.go:17]: Well I kind of need `gcloud` installed for other tests in a separate (and future) branch(es), so I'll leave it as-is, but nice pickup!

File: test/README.md
Comment (robmorgan) [reply test/README.md:40]: true!

File: .circleci/config.yml
Comment (robmorgan) [reply .circleci/config.yml:26]: ok sure.

**Comment by robmorgan** [comment]
This PR is blocked by https://github.com/gruntwork-io/gruntwork-installer/pull/58

**Comment by brikis98** [comment]
https://github.com/gruntwork-io/gruntwork-installer/pull/58 is merged & released.

**Comment by robmorgan** [comment]
Okay looks as if the tests passed in https://app.circleci.com/pipelines/github/gruntwork-io/terraform-google-ci/36/workflows/5aff1459-8bfd-4ac8-9194-f14e367a369c

**Review by bwhaley — APPROVED** [review]
LGTM

**Comment by robmorgan** [comment]
Reminder: only delete the branch once Terratest has been updated to use the release version.

**Pull request merged**
